portfolio/src/main/webapp/script.js

```diff
@@ -45,21 +84,50 @@ function login() {
 /* 
  * When asyncRequest is done, display the servlet response on the DOM
  */ 
-function stateChange() {
+function stateChangeFP() {
   if (asyncRequest.readyState == 4 && asyncRequest.status == 200) {
-    var toWrite = document.getElementById("DOMDisplay");
+    var homePageMsg = document.getElementById("DOMDisplay");
     var commentSubmissionForm = document.getElementById("comment-submission-form");
+    var deleteAllbtn = document.getElementById("delete-all");
     // If user is logged in, display comment submission form
     if (asyncRequest.responseText.toString().includes("You are logged in")) {
```

Comment:
Just FYI, you don't need make change. This is not a usual way to check whether your async is successful or not. Usually, it should return json or gson with defined status, e.g `{status: 200, message: whatever_error_or_return_obj}`

Reply:
Alright, thank you!